src/applications/check-in/pre-check-in/tests/e2e/update-skip-path/update.all.path.cypress.spec.js

```diff
@@ -0,0 +1,58 @@
+import '../../../../tests/e2e/commands';
+
+import ApiInitializer from '../../../../api/local-mock-api/e2e/ApiInitializer';
+import ValidateVeteran from '../../../../tests/e2e/pages/ValidateVeteran';
+import Introduction from '../pages/Introduction';
+import Confirmation from '../pages/Confirmation';
+import Demographics from '../../../../tests/e2e/pages/Demographics';
+import NextOfKin from '../../../../tests/e2e/pages/NextOfKin';
+import EmergencyContact from '../../../../tests/e2e/pages/EmergencyContact';
+
+describe('Pre Check In Experience', () => {
+  describe('update skip path', () => {
+    beforeEach(function() {
+      const now = Date.now();
+      const today = new Date(now);
+      const {
+        initializeFeatureToggle,
+        initializeSessionGet,
+        initializeSessionPost,
+        initializePreCheckInDataGet,
+        initializePreCheckInDataPost,
+      } = ApiInitializer;
+      initializeFeatureToggle.withCurrentFeatures();
+      initializeSessionGet.withSuccessfulNewSession();
+      initializeSessionPost.withSuccess();
+      initializePreCheckInDataGet.withSuccess({
+        demographicsNeedsUpdate: true,
+        demographicsConfirmedAt: today.toISOString(),
+        nextOfKinNeedsUpdate: true,
+        nextOfKinConfirmedAt: today.toISOString(),
+        emergencyContactNeedsUpdate: true,
+        emergencyContactConfirmedAt: today.toISOString(),
+      });
+      initializePreCheckInDataPost.withSuccess();
+
+      cy.visitPreCheckInWithUUID();
+      ValidateVeteran.validatePageLoaded();
+      ValidateVeteran.validateVeteran();
+      ValidateVeteran.attemptToGoToNextPage();
+      Introduction.validatePageLoaded();
+      Introduction.attemptToGoToNextPage();
+    });
+    afterEach(() => {
+      cy.window().then(window => {
+        window.sessionStorage.clear();
+      });
+    });
+    it('update demographics, next of kin path, and emergency contact', () => {
+      Demographics.validatePageLoaded();
```

Comment:
Address this warning ^ by [adding a axe check](https://depo-platform-documentation.scrollhelp.site/developer-docs/A11y-Testing.1935409178.html)